pkg/httpclient/httpclient.go

```diff
@@ -0,0 +1,30 @@
+package httpclient
```

Comment:
I'd prefer this be called `httputil` so that more http helpers could be put here in the future.

You should also include a package doc comment.